Author: <peter@p12n.org>
Description: Prevent recursive linking of dependent libraries by apr-util users.
Index: apr-util-1.3.12/apr-util.pc.in
===================================================================
--- apr-util-1.3.12.orig/apr-util.pc.in	2008-05-24 01:27:59.000000000 +0400
+++ apr-util-1.3.12/apr-util.pc.in	2011-05-24 16:22:48.122323498 +0400
@@ -8,6 +8,7 @@
 Description: Companion library for APR
 Version: @APRUTIL_DOTTED_VERSION@
 # assume that apr-util requires libapr of same major version
-Requires: apr-@APRUTIL_MAJOR_VERSION@
-Libs: -L${libdir} -l@APRUTIL_LIBNAME@ @LDADD_ldap@ @APRUTIL_EXPORT_LIBS@
+Requires.private: apr-@APRUTIL_MAJOR_VERSION@
+Libs: -L${libdir} -l@APRUTIL_LIBNAME@ @LDADD_ldap@
+Libs.private: @APRUTIL_EXPORT_LIBS@
 Cflags: -I${includedir}
Index: apr-util-1.3.12/apu-config.in
===================================================================
--- apr-util-1.3.12.orig/apu-config.in	2011-05-24 16:22:36.302660081 +0400
+++ apr-util-1.3.12/apu-config.in	2011-05-24 16:22:48.128643471 +0400
@@ -27,7 +27,7 @@
 libdir="@libdir@"
 includedir="@includedir@"
 
-LIBS="@APRUTIL_EXPORT_LIBS@"
+LIBS=
 INCLUDES="@APRUTIL_INCLUDES@"
 LDFLAGS="@APRUTIL_LDFLAGS@"
 ORIG_LDAP_LIBS="@LDADD_ldap@"
